components/salesforce_rest_api/README.md

```diff
@@ -4,6 +4,53 @@ You can install the Pipedream Salesforce app in the [Accounts](https://pipedream
 
 ### Webhooks
 
+The following three permissions must be configured on your Salesforce user in order to use the Salesforce Instant (webhook) triggers:
+1. Apex REST Services
+2. API Enabled
+3. Author Apex (which adds in additional permissions.)
+
+The most straightforward way to add these permissions is to create a new Permission Set, and to add it to the user once created.
+
+Here is a step-by-step on how to do this:
+
+**Create New Permission Set**
+1. Under the Administration tab on the lefthand sidebar, click Users --> Permission Sets.
+2. On the Permissions Set page, click New.
+3. Create a new permission set, give it a label, API name, and description. Example:
+**Label**: Pipedream API Access
+**API Name**: Pipedream
+**Description**: Adds a set of permissions required for Pipedream sources:
+- Apex REST Services
+- API Enabled
+- Author Apex
+
+ <img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598220/Screenshot_2023-12-14_at_2.57.21_PM_dfgsrw.png" width=500>
+
+**Add Permissions**
+
+4. Now that the permission set is created, navigate to System Permissions.
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598358/Screenshot_2023-12-14_at_3.00.49_PM_axtws5.png" width=500>
+5. From System Permissions, click Edit.
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598417/Screenshot_2023-12-14_at_3.01.38_PM_pvbopv.png" width=500>
+
+6. Select the following permissions, and click Save.
+- Apex REST Services
+- API Enabled
+- Author Apex (which adds in additional permissions.)
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598514/Screenshot_2023-12-14_at_3.48.50_PM_pcychy.png" width=500>
+
+7. The list of added permissions should look like this, and click save again.
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598417/Screenshot_2023-12-14_at_3.10.17_PM_urgge8.png" width=500>
+
+**Add Permission Set to User**
+8. From the newly created Permission Set, click Manage Assignments, then Add Assignment.
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598514/Screenshot_2023-12-14_at_3.21.59_PM_rqedtd.png" width=500>
+
+9. Select the user you'd like to assign this permission set to, and click Assign. The user should now show up under Current Assignments.
+<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1702598514/Screenshot_2023-12-14_at_3.52.42_PM_w4ge4p.png" width=500>
+
+### Troubleshooting
```

Comment:
is this a child of the new instructions you're adding? if not, I'd make H1 so it renders in its own linkable section.